modules/imaging_uploader/jsx/UploadForm.js

```diff
@@ -81,6 +81,34 @@ class UploadForm extends Component {
     }
 
     const fileName = data.mriFile.name;
+    // Make sure file is of type .zip|.tgz|.tar.gz format
+    const properExt = new RegExp('.(zip|tgz|tar.gz)$');
```

Comment:
The "."s in this regex aren't escaped, "." means "any character" in a regular expression not a literal "."

Reply:
Is this fixed by a backslash?

```suggestion
    const properExt = new RegExp('\.(zip|tgz|tar.gz)$');
```

I'm not sure good with JS

Reply:
yes, but you're missing the one in tar.gz and I think it needs to be retested

Reply:
@cmadjar

Reply:
right. I saw that and meant to correct it but I guess I was distracted ;). Done in the last commit